src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -801,11 +811,30 @@ public function testPutState()
         $this->assertEquals(1, $response['nodeState']);
     }
 
+    private function import($file)
+    {
+        if ($this->session->getRootNode()->hasNode('cmf')) {
+            $this->session->getNode('/cmf')->remove();
+            $this->session->save();
+        }
+
+        $this->session->importXML(
+            '/',
+            __DIR__ . '/../../app/Resources/exports/' . $file . '.xml',
+            ImportUUIDBehaviorInterface::IMPORT_UUID_COLLISION_REPLACE_EXISTING
```

Comment:
Wouldn't it be saver to not set this? Then we would realize if something is going wrong with the purging, which is needed for a clean test.